samples/net/sockets/mtls_http_get/README.rst

```diff
@@ -0,0 +1,86 @@
+.. _sockets-http-get:
+
+Socket HTTP GET Example with mTLS
+#################################
+
+Overview
+********
+
+The sockets/http_get sample application for Zephyr implements a simple
+HTTP GET client using a BSD Sockets compatible API and mutual TLS
+authentication. The purpose of this sample is to show how it's possible
+to do mTLS with Zephyr. 
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/http_get`.
```

Comment:
This should be fixed for the directory we're in now. Maybe reference where it came from?